### ISSUE TITLE
Improve tab index on unit field

### ISSUE THREAD

**Issue by fabianhueske** [shopware/meteor#829]
### Package

@meteor/component-library

### Steps to reproduce

Use unit field with keyboard controls.

### Actual behavior

Up/down buttons get focused first, then unit select

### Expected behavior

Unit select should be focused before the up/down buttons
